packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -495,3 +500,48 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
     properties.add(EnumProperty<GestureRecognizerState>('state', state));
   }
 }
+
+/// A container for a [local] and [global] [Offset].
+///
+/// Usually, the [global] [Offset] is in the coordinate space of the screen and
+/// the [local] offset is the same [Offset], but transformed to a local
```

Comment:
actually isn't [global] usually in the screen coordinate system _after conversion to logical pixels_?